suite-common/trading/src/thunks/common/recomposeAndSignTxThunk.ts

```diff
@@ -75,6 +78,8 @@ export const recomposeAndSignTxThunk = createThunk<
         const { composed, selectedFee } = selectTradingComposedTransactionInfo(getState());
         const options: FormOptions[] = ['broadcast'];
         const network = getNetwork(account.symbol);
+        const firmwareVersion = selectDeviceFirmwareVersion(getState());
+        const isNewApproveFlow = firmwareVersion && isNewerOrEqual(firmwareVersion, '2.9.0');
```

Comment:
Is there a reason for calling it `New`?
```suggestion
        const isApproveFlow = firmwareVersion && isNewerOrEqual(firmwareVersion, '2.9.0');
```